bin/export_mc.py

```diff
@@ -0,0 +1,185 @@
+#!.venv/bin/python
+
+import argparse
+import os
+import subprocess
+import sys
+from git import Repo
+
+DEFAULTS = {
+    "local": "http://localhost:10080/",
+    "dev": "https://pageshot.dev.mozaws.net/",
+    "stage": "https://pageshot.stage.mozaws.net/",
+    "prod": "https://pageshot.net"
```

Comment:
not sure if it matters, but prod doesnt have a trailing `/`, but the other envs do.